__tests__/components/AnonymousChat/child/ChatItemMyTextV2.test.tsx

```diff
@@ -56,6 +57,12 @@ beforeEach(() => {
   setupTests();
 });
 
+jest.mock('react-native-compressor', () => {
+  return {
+    compress: jest.fn(() => 'file:///imag.jpg')
+  };
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Mocking the `react-native-compressor` module is a good practice for unit tests. However, it's important to ensure that the mock function returns a value that matches the actual function's return type. In this case, the `compress` function should return a Promise according to the library's documentation. Please update the mock to return a Promise.
```diff
-    compress: jest.fn(() => 'file:///imag.jpg')
+    compress: jest.fn(() => Promise.resolve('file:///imag.jpg'))
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->